src/meta/topics/building-from-src.md

```diff
@@ -0,0 +1,198 @@
+## Building from src
```

Comment:
We might --headingLevel throughout this document — this should be `# Building From Source`.